### PR TITLE
Reload on Save feature

### PR DESCRIPTION
**Pull request by gcb**
my usage of uMatrix is to block everything by default, so my usage is:

1. load a site all broken because everything is blocked
1. click the extension button
1. allow local scripts or something else.
1. click the padlock
1. click the reload
1. if site is still broken, go back to step 2

with this new setting enabled, i don't have to click the padlock and the reload button (which are not that close). I just click the padlock, see the site reload behind the popup and either dismiss it or continue to fiddle.

 
I took the liberty to include the makefile i was using (so new developers can simply type `make` and have a build ready to use)

### CODE REVIEW

**Comment by gcb** [comment]
added translations for the languages i know. (though i am on beginner level for Japanese and Chinese, someone might want to double check those)

File: Makefile [hunk 1]
```diff
@@ -4,8 +4,12 @@ PHONY: clean
 clean:
 	-rm -r dist/
 
-PHONY: build
-build:
-	tools/make-firefox.sh
+build: dist/build/uMatrix.chromium dist/build/uMatrix.firefox
```
Comment (ruv) [inline Makefile]: Actually `build`  should be marked as PHONY target since it is not a file name.

**Comment by qazip** [comment]
Thank you so much for this. In what version of umatrix is/will this be available? Do you know?

**Comment by gorhill** [comment]
I work differently: I set rules and reload. Once the page works, I click the padlock. A reload-on-save would be a nuisance to me and all other working the same way.

**Comment by gcb** [comment]
@gorhill I added reloadOnSave as an option, disabled by default.

File: Makefile (hunk 1 above)
Comment (gcb) [reply Makefile]: /me facepalm
will add back if reopen

**Comment by elypter** [comment]
it would be cool if you could create a fork that merges all the pull requests gorhill declined. it's starting to become worth it and it would create a place where people could post other requests as well.

**Comment by qazip** [comment]
I agree, someone should fork umatrix and enable auto-reload. It's the one feature keeping me on noscript.

**Comment by elypter** [comment]
i havent tested it yet but here is a fork with some of the pull requests merged https://github.com/elypter/uMatrix
it also merges a few changes for the user agent switcher that got removed just minutes ago

**Comment by qazip** [comment]
@elypter, you should re-do the readme to show that it's a fork and you should also mention the advantages of your version over gorhil's. And ofcourse, publish it on firefox' addons page.

**Comment by gcb** [comment]
feel free to merge wherever. It is a very easy patch to apply (only 4 lines of actual code change. Thanks gorhill for such a tidy code base!)

on the UI translations i cannot guarantee for chinese. i am barely a beginner.

**Comment by elypter** [comment]
i have zero expierience with firefox addons. i just clicked through github and merged a few patches. the result doesnt work yet. i will change the readme if it becomes useable. i wished github would be more helpful with that kind of stuff

**Pull request closed without merging**